plugins/akka-http/src/main/java/com/navercorp/pinpoint/plugin/akka/http/interceptor/ExecutionDirectivesHandleExceptionsInterceptor.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2017 NAVER Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.navercorp.pinpoint.plugin.akka.http.interceptor;
+
+import akka.http.javadsl.model.HttpRequest;
+import akka.http.scaladsl.server.RequestContextImpl;
+import com.navercorp.pinpoint.bootstrap.async.AsyncContextAccessor;
+import com.navercorp.pinpoint.bootstrap.context.*;
+import com.navercorp.pinpoint.bootstrap.interceptor.AroundInterceptor;
+import com.navercorp.pinpoint.bootstrap.logging.PLogger;
+import com.navercorp.pinpoint.bootstrap.logging.PLoggerFactory;
+import com.navercorp.pinpoint.bootstrap.plugin.proxy.ProxyHttpHeaderHandler;
+import com.navercorp.pinpoint.bootstrap.plugin.proxy.ProxyHttpHeaderRecorder;
+import com.navercorp.pinpoint.plugin.akka.http.AkkaHttpConstants;
+
+import java.util.Optional;
+
+public class ExecutionDirectivesHandleExceptionsInterceptor implements AroundInterceptor {
+
+    private final PLogger logger = PLoggerFactory.getLogger(ExecutionDirectivesHandleExceptionsInterceptor.class);
+    private final boolean isDebug = logger.isDebugEnabled();
+    private final TraceContext traceContext;
+    private final MethodDescriptor descriptor;
+    private static final AkkaHttpServerMethodDescriptor AKKA_HTTP_SERVER_METHOD_DESCRIPTOR = new AkkaHttpServerMethodDescriptor();
+    private final ProxyHttpHeaderRecorder proxyHttpHeaderRecorder;
+
+    public ExecutionDirectivesHandleExceptionsInterceptor(final TraceContext traceContext, final MethodDescriptor methodDescriptor) {
+        this.traceContext = traceContext;
+        this.descriptor = methodDescriptor;
+        this.proxyHttpHeaderRecorder = new ProxyHttpHeaderRecorder(traceContext.getProfilerConfig().isProxyHttpHeaderEnable());
```

Comment:
`traceContext.cacheApi(AKKA_HTTP_SERVER_METHOD_DESCRIPTOR);`
to cache method descriptor is required